react/features/notifications/components/Notification.web.js

```diff
@@ -83,19 +148,117 @@ class Notification extends Component {
 
         return (
             <Flag
-                appearance = 'normal'
-                description = { t(descriptionKey, descriptionArguments) }
-                icon = { (
-                    <EditorInfoIcon
-                        label = 'info'
-                        size = 'medium' />
-                ) }
+                actions = { this._mapAppearanceToButtons(addSupportLink) }
+                appearance = { appearance }
+                description = { description
+                    || t(descriptionKey, descriptionArguments) }
+                icon = { this._mapAppearanceToIcon() }
                 id = { uid }
                 isDismissAllowed = { isDismissAllowed }
                 onDismissed = { onDismissed }
-                title = { title || t(defaultTitleKey) } />
+                title = { title || t(titleKey) } />
         );
     }
+
+    /**
+     * Calls back into {@code FlagGroup} to dismiss the notification.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onDismissed() {
+        this.props.onDismissed(this.props.uid);
+    }
+
+    /**
+     * Opens the support page.
+     *
+     * @returns {void}
+     * @private
+     */
+    _onOpenLink() {
+        window.open(interfaceConfig.SUPPORT_URL, '_blank');
```

Comment:
Does window.open need to be called with a noreferrer option in the argument to prevent the url from have accessing to the opener?